widgets/LiquidContainer/LiquidContainer.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef LIQUID_CONTAINER_HPP
+#define LIQUID_CONTAINER_HPP
+
+#include <mvp/View.hpp>
+#include <gui/template_screen/TemplatePresenter.hpp>
+#include <touchgfx/widgets/Image.hpp>
+#include <BitmapDatabase.hpp>
+#include <touchgfx/containers/Container.hpp>
+#include <touchgfx/widgets/TextAreaWithWildcard.hpp>
+#include <texts/TextKeysAndLanguages.hpp>
+
+using namespace touchgfx;
+
+class LiquidContainer : public Container
```

Comment:
We'd like to change the name of the container to "AnimatedContainer" (that in this case just happens to use imags of liquid). Names of members and methods should be modified in accordance with the new name of the widget (i.e. liquidImageId, liquid, liquid2, etc.)